src/foam/nanos/session/services/LocalSettingsService.js

```diff
@@ -0,0 +1,19 @@
+foam.INTERFACE({
```

Comment:
This service is for setting/getting a particular settings.
so two options.
LocalSetting is an model of 
id String
value String
and then this is just a DAO.
The client does normal DAO puts on startup. 
The client can also select from the server and do a sync - update client with those on server and update server with those on client. 

If you go the model/DAO route then you don't need this 'service', just a couple DAO decorators for client/server and something on the client side (might be something in EasyDA) - which runs when the DAO is instantiated.

Reply:
are we gonna store homeDenomination in DAO? I thought we just need to use it for homeBalance calculation.

or is it a DAO to store the name of local settings we set in Session Context?

Reply:
This is about all localStorage properties, homeDenomination is just one of many.
The DAO is localSetting - id,value pair. 
The id in this case is 'homeDenomination' and the value is whatever it needs to be. 
Retrieving - two options: 
1. explicitly query localSettingDAO for some id 
2. place all localSetting entries into the Session, the lookup via context key.

Reply:
But we don't want the home denomination to be global for all of a user's logins, since they could have a different one for each device they're logged into. Would it just be a transient DAO in their context?

Reply:
yes, all transient. 
The dao can be in their session or each entry can be added to the session (context). 
To deal with Chrome deleting localStorage, we can store a device id with the setting if we want to store for the user (permanently). 
So when the client instance starts it loads localStorage then sends all to the server - and those entries are added to the users localSettingDAO- in their session context. 
And future, when localStorage is empty, the client can query for last saved settings for that device and recreate localStorage.